docs/High-Level_Design.md

```diff
@@ -0,0 +1,112 @@
+
+### Example forms and project list
+[Register new case/study](https://liveuclac.sharepoint.com/sites/ISD.IGAdvisoryService/Lists/Start%20a%20service%20request/NewForm.aspx)
+
+[View my projects](https://liveuclac.sharepoint.com/sites/ISD.IGAdvisoryService/Lists/Cases/Portal.aspx)
+
+## To-be Portal
+
+![High-level view](./media/architecture.png)
+
+The replacement portal will manage [key ISMS processes](https://github.com/UCL-ARC/research-data-isms/tree/main/docs/Controlled_Processes). This portal will be use the pathfinder project for ARC's metadata store based on
+[this design](https://github.com/UCL-ARC/metadata-store/blob/main/doc/specification.md).
+Some processes such as incident and service request will be managed through "MyServices" (Xurrent) as per the pattern detailed [here in option 2](https://github.com/UCL-ARC/ARC-Strategy-and-Design/blob/main/Docs/Service_Design/ARC_Experience_and_Service_Platforms.md).
+
+## Design Principles
+
+Design will adhere to the [digital foundation principles](https://liveuclac.sharepoint.com/sites/it-architecture/SitePages/digital-foundation-principles.aspx).
+
+Plus:
+
+| Principle | Description | Implementation | Testing |
+|------------|------------|----------------|----------|
+|Reusable | The portal and underlying data structure will support a wide variety of use cases within ARC, initially focused on ARC services requiring similar functionality (EG. User onboarding) | Initially built around the TRE use case with input from the Research Computing and Research Data team to ensure designs support other use cases. | TBC
+| Open Source | As much of the development should be made open source as possible with an output of the build being a design and code base. | TBC | TBC
+| Master state data in the right pace | The state data of services and resources will remain mastered within platforms. This will include access control to resources and configuration of infrastructure. These state data will be read from the platforms (Such as TRE, RDP and Condenser) and processes managed within the portal may trigger changes in state within the platform. | TBC | TBC
+| Retention of Quality Data | Where processes within the portal make changes to metadata or resources adequate quality data will be captured and retained in line with the requirements of any downstream system. | TBC | TBC
+
+## Non-functional Requirements
+
+### Data Classification
+
+The portal and underlying data will be limited to storing confidential and public data.
+
+### Availability
+
+TBC
+
+### Integrity
+
+See above principle on quality data.
+
+## Definition of the MVP
+
+
+
+## Conceptual data model
+This model represents an overview of the current understanding of the target main entities referenced in the project stories, with minimal detail. It improves in some respects from the model in the [as-is Portal](#as-is-portals).
+
+```mermaid
```

Comment:
for me this goes beyond what's needed for the portal. why are we recording environment owners for instance?

Reply:
hm, I forget. @jackhindleyucl ?

Happy to remove it.